### PR TITLE
add example for offline use

### PR DESCRIPTION
**Pull request by lalo**

### CODE REVIEW

File: coba/environments/filters.py [hunk 1]
```diff
@@ -997,6 +997,8 @@ def filter(self, interactions: Iterable[Interaction]) -> Iterable[Interaction]:
                 new['actions'] = list(map(methodcaller('copy'),new['actions']))
 
             if not action_materialized and is_dense_action:
+                #new['action'] = list(new['action'])
```
Comment (lalo) [inline coba/environments/filters.py:1000]: related to this previous PR - https://github.com/VowpalWabbit/coba/pull/24

Comment (lalo) [reply coba/environments/filters.py:1000]: https://github.com/VowpalWabbit/coba/actions/runs/4128674424/jobs/7133406460#step:10:131

**Comment by lalo** [comment]
LGTM, thanks @mrucker

**Comment by mrucker** [comment]
@lalo Just released the bug patch with version 6.2.8

**Pull request merged**